src/hansel.js

```diff
@@ -8,7 +9,7 @@ export const HANDLER_ATTRIBUTE = 'data-handler';
  */
 export const enhance = (root, enhancers) => {
   if (!enhancers) {
-    return;
+    return null;
```

Comment:
I would like to return an empty array here. That would be consistent with the return type in case there _are_ `enhancers`.

Reply:
Sure, it makes sense !